### ISSUE TITLE
multipart/form-data implementation

### ISSUE THREAD

**Issue by mission-liao** [pyopenapi/pyswagger#1]
Tend to provide an implementation. However, the encoding spec of filename, field name with non-ASCII chars is really a mess.

At first, I think the right way is to apply **RFC2047**, but it states this algorithm can't apply on a "a MIME Content-Type or Content-Disposition field". Then the second spec I found is **RFC2231**. And I plan to do it until I finally found another [SO question](http://stackoverflow.com/questions/3393445/international-characters-in-filename-in-mutipart-formdata).

In short, they suggested to send those names, attributes in **utf-8**, which is not official but supported by most major web servers, because almost all browsers did this part in this way...

After all this survey, now I feel ready for multipart/form-data.
https://github.com/AntXlab/pyswagger/labels
